kitsune/sumo/static/sumo/scss/base/forms/_fields.scss

```diff
@@ -411,5 +411,18 @@ textarea {
 label.required::after {
   content: "\A Required";
   white-space: pre;
-  color: var(--color-marketing-red-01);
+  display: block;
+  font-size: 14px;
+  font-size: .875rem;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: 0;
+}
+.mzp-c-fieldnote {
```

Comment:
I think we should rename this class because it's not the same as Protocol's `mzp-c-fieldnote`. I think it's only used in `kitsune/questions/jinja2/questions/includes/question_editing_frame.html`?

Reply:
I made custom.